packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-modal-foot-note.tsx

```diff
@@ -80,6 +80,8 @@ const JurisdictionModalFootNote = (props: TJurisdictionModalFootNoteProps) => {
                 <FooterNote {...props} />
             </Text>
         </div>
+    ) : (
+        <React.Fragment />
```

Comment:
What is this for?

Reply:
@matin-deriv in mobile view, an empty container is being displayed when no card is selected. so if no card is selected i thought of returning an empty fragment instead of rendering empty div/Text which has some stylings. is this fine? or shall i write some other logic to prevent the component from being rendered?
<img width="474" alt="Screenshot 2023-04-06 at 9 04 31 AM" src="https://user-images.githubusercontent.com/122449658/230276802-d487ddbf-0d34-4f43-9567-aaf82e285cc1.png">

Reply:
I think we need to remove the empty container instead of leaving empty fragment

Reply:
when i write something like `props.selected && <Component />`, typescript says the component cannot be used since it may return empty string in false scenario. 
so shall i write a Fragment wrapper inside which this condition+component is written, like this?
```
<Fragment>
       {props.selected && <Component />}
</Fragment>
```
<img width="608" alt="Screenshot 2023-04-10 at 10 08 26 AM" src="https://user-images.githubusercontent.com/122449658/230838106-2357b228-88c1-49bb-83aa-74485f0a1dc4.png">